openstack/clustering/v1/policies/requests.go

```diff
@@ -118,3 +119,13 @@ func Create(client *gophercloud.ServiceClient, opts CreateOpts) (r CreateResult)
 	})
 	return
 }
+
+// Create makes a request against the API to delete a policy
+func Delete(client *gophercloud.ServiceClient, policyID string) (r PolicyDeleteResult) {
```

Comment:
This function _should_ be able to be simplified a little:

https://github.com/gophercloud/gophercloud/blob/master/openstack/objectstorage/v1/containers/requests.go#L115-L119

However, keep the `OkCodes` part. 

I haven't tested this myself, just going off of similar existing functions.